_test/test_utilities_herbert/test_object_lookup_internals.m

```diff
@@ -0,0 +1,195 @@
+classdef test_object_lookup_internals < TestCase
+    % Test of utility functions e.g. for parsing that are defined
+    % in the @object_lookup folder
+
+    methods
+        %--------------------------------------------------------------------------
+        function obj = test_object_lookup_internals (name)
+            obj@TestCase(name);
+        end
+        
+        %--------------------------------------------------------------------------
+        function test_parse_split_first_element_only (~)
+            % Test first element true only
+            split = exposed_object_lookup.parse_split (2, 'split', 1);
+            assertEqual (split, [true, false]);
+        end
+        
+        function test_parse_split_second_element_only (~)
+            % Test second element true only
+            split = exposed_object_lookup.parse_split (2, 'spl', 2);
+            assertEqual (split, [false, true]);
+        end
+        
+        function test_parse_split_all_elements (~)
+            % Test all elements true
+            split = exposed_object_lookup.parse_split (3, 'spl');
```

Comment:
Not 100% sure I like the fuzzy matching (i.e. 'sp' == 'split'), but I would make that a separate issue.